lib/internal/console/constructor.js

```diff
@@ -421,7 +421,10 @@ const consoleMethods = {
   trace: function trace(...args) {
     const err = {
       name: 'Trace',
-      message: this[kFormatForStderr](args)
+      message: this[kFormatForStderr](args),
+      toString() {
+        return `${this.name}: ${this.message}`;
+      }
```

Comment:
would things be faster if this method were shared instead of created every time trace is called? (perhaps memoized on first call)

Reply:
I am not certain I am able to follow. I would expect that the trace call is not used in performance sensitive code and that it'll mainly log unique values (the error message could be different on each call, even if it's the same line of code that triggers it). As such, memoizing this would probably not bring a benefit and only cause memory to fill up?

Reply:
i don't mean memoizing the string output - i just mean this one function, whose code is the same every time, can be reused rather than created fresh every time

Reply:
Ah, yes, it would be a tad faster while likely negligible. I consider the readability more important in this case.

Reply:
I would expect that too, but this entire PR assumes that performance of the error path is worth improving, no?